src/pages/achievements/[id].tsx

```
@@ -96,14 +101,60 @@
 
+  const mintable = useMemo(() => {
+    return achievement?.community?.can_mint_certificates || isICPSubmission;
+  }, [achievement, isICPSubmission]);
+
   const isNotCertificateIcon = useMemo(() => {
     return !achievement?.metadata?.image?.includes("/img/certificates/");
   }, [achievement]);
 
+  const requestVC = (principal: string) => {
+    if (!achievement?.metadata) return;
+    const { issuedOn, linkToWork, issuerName, name, narrative, image, comment } = achievement?.metadata;
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)